src/main/java/nl/rijksoverheid/mev/gezagsmodule/service/newversion/GezagBepaling.java

```diff
@@ -93,4 +95,62 @@ public Persoonslijst getPlNietOuder() {
 
         return plNietOuder;
     }
+
+
+    /**
+     * @return of er velden in onderzoek waren (010120 en 080910 worden
+     * gefiltered)
+     */
+    public boolean warenVeldenInOnderzoek() {
+        List<String> veldenInOnderzoek = plPersoon.getUsedVeldenInOnderzoek();
+        if (plOuder1 != null) {
+            veldenInOnderzoek.addAll(plOuder1.getUsedVeldenInOnderzoek());
+        }
+
+        if (plOuder2 != null) {
+            veldenInOnderzoek.addAll(plOuder2.getUsedVeldenInOnderzoek());
+        }
+
+        if (plNietOuder != null) {
+            veldenInOnderzoek.addAll(plNietOuder.getUsedVeldenInOnderzoek());
+        }
+
+        if (!veldenInOnderzoek.isEmpty()) {
+            log.info("Velden waren in onderzoek: {}", veldenInOnderzoek);
```

Comment:
Dit kan een algemene info zijn zonder if statement erom heen, e.g.

"De volgende velden zijn in onderzoek: []"

Verder zou ik tegenwoordige tijd gebruiken (zijn vs. waren) omdat we volgens mij alleen maar werken in de tegenwoordige tijd.